### PR TITLE
Improve the description and add login link

### PR DESCRIPTION
**Pull request by arturogonzalez58**
This PR improves the main action description, sets the color for the action in the marketplace, and adds the link to login into app.getambassador.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Telepresence combined with GitHub Actions allows you to run integration tests in
 
 ## V1.0
 
-The V1.0 of github actions for Telepresence contains individual actions to:
+The V1.0 of GitHub actions for Telepresence contains individual actions to:
```
Comment (hojenkai) [inline README.md:7]: v1.0.0 ?

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Telepresence combined with GitHub Actions allows you to run integration tests in
 
 ## V1.0
```
Comment (hojenkai) [inline README.md:5]: v1.0.0 ?

**Review by hojenkai — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 Telepresence combined with GitHub Actions allows you to run integration tests in your CI pipeline without having to run any dependant service. By connecting to the target Kubernetes cluster, intercepting traffic to the remote service and sending it to an instance of the service running in CI you will be able to test bugfixes, updates and features very easily.
 
-## V1.0
+## v1.0.0
 
-The V1.0 of github actions for Telepresence contains individual actions to:
+The v1.0.0 of GitHub actions for Telepresence contains individual actions to:
```
Comment (khussey) [inline README.md:7]: Should be "Telepresence GitHub Actions" instead of "GitHub actions for Telepresence".

File: README.md [hunk 4]
```diff
@@ -13,10 +13,13 @@ The V1.0 of github actions for Telepresence contains individual actions to:
 1. [Intercept](./intercept) traffic of service running in the K8s cluster and redirect it a service instance running in CI.
 
 See the [Telepresence GitHub actions documentation page](https://www.getambassador.io/docs/telepresence/latest/ci/github-actions/) for information about how to use these actions and build an integration tests workflow for your repository.
-**Note**: The version v1.0 only supports Ubuntu runners at moment.
+**Note**: The version v1.0.0 only supports Ubuntu runners at moment.
 
 ## Usage
+### Prerequisites
+Get an ambassador free account. Register [here](https://app.getambassador.io/auth/realms/production/protocol/openid-connect/auth?client_id=telepresence-github-actions&response_type=code&code_challenge=qhXI67CwarbmH-pqjDIV1ZE6kqggBKvGfs69cxst43w&code_challenge_method=S256&redirect_uri=https://app.getambassador.io) to get a free account. No credit card required.
```
Comment (khussey) [inline README.md:20]: Should be "Get a free Ambassador Cloud account."

**Review by khussey — CHANGES_REQUESTED** [review]
Minor corrections.

**Review by khussey — APPROVED** [review]
+1

**Pull request merged**